docs/src/api/class-worker.md

```diff
@@ -38,14 +38,14 @@ for worker in page.workers:
 ```
 
 ```csharp
-Page.Worker += (_, worker) =>
+page.Worker += (_, worker) =>
```

Comment:
This change is for consistency. I used lowercase in all the samples across the doccs.